src/arch/wasm32/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Could this be expanded a bit with a short comment saying that whenever LLD changes we need to rerun this script and recommit the archive?

It may also be worth calling out that `libstacker.a` is actually an object file, so the extension is lying a little bit

Reply:
Added in README.md!